packages/manager/modules/pci/src/projects/project/storages/databases/database/pools/terminate/terminate.module.js

```diff
@@ -0,0 +1,14 @@
+import angular from 'angular';
+
+import component from './terminate.component';
+import routing from './terminate.routing';
+
+const moduleName = 'ovhManagerPciStoragesDatabasePoolsTerminate';
+
+angular
+  .module(moduleName, [])
```

Comment:
**nitpick:** Following module is using `$translate` and `oui-modal`. These dependencies should be listed as a module dependencies.